src/daemon/protocols/edp.c

```diff
@@ -363,6 +367,7 @@ edp_decode(struct lldpd *cfg, char *frame, int s, struct lldpd_hardware *hardwar
 			PEEK_DISCARD_UINT16; /* vchassis */
 			PEEK_DISCARD(6);     /* Reserved */
 			PEEK_BYTES(version, 4);
+			free(chassis->c_descr);
```

Comment:
Fix a memory leak.
A memory leak occurs when case `EDP_TLV_INFO` is called twice.

Scenario:
1- Call occurs.
-----> Memory leak.
2- Call occurs.
-----> Memory is freed by the program.

Therefore, if there are two or more calls to case `EDP_TLV_INFO`, all of them leak memory, except the last one.
This change handles all memory leaks.